config/settings.py

```diff
@@ -257,6 +257,8 @@
 # UA-XXXXXXXXX-X
 GOOGLE_TRACKING_ID = os.environ.get("GOOGLE_TRACKING_ID")
 
+IPFS_GATEWAY = os.environ.get("IPFS_GATEWAY", "https://ipfs.io/ipfs/")
```

Comment:
maybe we need anoother gateway soon

Reply:
> maybe we need another gateway soon

yes i made this a .env value   just add it to the .env and make it whatever you want.  
IPFS_GATEWAY=https://newgateway.com/ipfs/               
right now it defaults to ipfs.io/ipfs/ if no value is in .env